boards/arm/nucleo_l412rb_p/nucleo_l412rb_p.yaml

```diff
@@ -0,0 +1,18 @@
+identifier: nucleo_l412rb_p
+name: ST Nucleo L412RP-P
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 40
+flash: 64
+supported:
+  - nvs
```

Comment:
This is probably the reason why partitions are requested.
This is welcome, but could be removed if not needed